src/table/settings/gui_settings.ini

```diff
@@ -512,13 +512,13 @@ cat      = SC_EXPERT
 var      = gui.cycle_signal_types
 type     = SLE_UINT8
 flags    = SF_NOT_IN_SAVE | SF_NO_NETWORK_SYNC | SF_GUI_DROPDOWN
-def      = 0
-min      = 0
-max      = 1
+def      = SIGNAL_CYCLE_ALL
```

Comment:
This default seems odd to me. Wouldn't we want to restrict new players to only cycle through the current signals?

(well, they don't have block signals active, but still, say they do activate that, because of some rant on some forum they read).

Reply:
Sure, I can make the other option default. 🙂